### ISSUE TITLE
Update Nav examples text

### ISSUE THREAD

**Issue by evwilkin** [patternfly/patternfly#5008]
The navigation docs are throwing a11y violations because we are re-using the words 'Link 1', 'Link 2', etc across examples. Each of the links in the example needs to have unique text, too.

_Originally posted by @nicolethoen in https://github.com/patternfly/patternfly/pull/4928#discussion_r907416600_

**Comment by mcoker** [comment]
@nicolethoen @evwilkin what violation is that? I'm not seeing that in @nicolethoen's PR that runs the tests against the component examples - https://github.com/patternfly/patternfly/pull/5064

I'm also not seeing it when I run axe locally - I just see the [identical-links-same-purpose violation](https://dequeuniversity.com/rules/axe/4.3/identical-links-same-purpose) triggered for
* skip to content - the actual component/button used on the workspace with the text "skip to content", and the "skip to content" link in the vertical nav to the component
* accessibility - the site header nav link to `/accessibility-guide`, and the "accessibility" jump link that takes you to `#accessibility`

**Comment by nicolethoen** [comment]
Yeah - that identical link text issue is (i believe) only a 'needs review' level of severity according to axe testing, so generally I have not worried about those as much.

**Comment by evwilkin** [comment]
Closing per comment above that 'needs review' severity issues aren't prioritized to fix at this time.
